generate-map.py

```diff
@@ -48,6 +50,42 @@ def getGeoCode(country, city, organization, dataList):
         return geoLocation
 
 
+def fixOverlappingEntries():
```

Comment:
Suggestion:

```python
locations = {}
for organization in data:
    key = (organization['longitude'], organization['latitude'])
    # check if we already encountered this location
    if locations.get(key):
        locations[key]['organization'] += ', ' + organization['organization']
    else:
        locations[key] = organization
return locations.values()
```

Reply:
I added the suggestions. Your way is much more easy and clear. thank you! :D